### PR TITLE
Sentiment Analysis: Step #1

### PR DESCRIPTION
**Pull request by TengkuHanaShafikah**
I apologize for how confusing these particular set of commits are — I went through a bit of a pickle with git, and I decided to commit these changes first before I dig a deeper hole for myself. 😅 Below are the general changes I made in this PR:

- **pom.xml:** Added Cloud Natural Language dependency to pom.xml file
- **Comment.java:** Modified Comment class to have new attribute for the sentiment score.
- **script.js:** Added JavaScript code that displays sentiment score in the comments.

My next steps will be to modify DataServlet.java to calculate the sentiment score of the comment entered by the user, add the sentiment score to your comment storage, then add the sentiment score to the JSON returned by the servlet.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -50,11 +48,15 @@ function createListElement(comment) {
   const textElement = document.createElement('span');
   textElement.innerText = comment.text;
 
+  const scoreElement = document.createElement('span');
+  scoreElement.innerText = " (Score: " + comment.score + " | ";
+
   const timeElement = document.createElement('span');
-  timeElement.innerText = "   (" + comment.date + ")";
+  timeElement.innerText = "Date: " + comment.date + ")";
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js]: Try using [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)!

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 2]
```diff
@@ -7,12 +7,14 @@
   private final long id;
   private final String text;
   private final String name;
+  private final float score;
   private final Date date;
 
-  public Comment(long id, String name, String text, long time) {
+  public Comment(long id, String text, String name, float score, long time) {
```
[the remark this answers is not in the document]
Comment (TengkuHanaShafikah) [reply portfolio/src/main/java/com/google/sps/data/Comment.java]: I've checked and it's consistent with what's in `DataServlet.java`! 👍

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -33,28 +33,30 @@ function getComments() {
...
-      });
+      comments.map(createListElement).forEach(line => commentsListElement.appendChild(line));
     });
 }
 
-/** Creates an <li> element containing comment data.*/
+/** Creates an <ul> element containing comment data.*/
 function createListElement(comment) {
-  const commentElement = document.createElement('li');
+  const commentElement = document.createElement('ul');
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js]: Should it be `ul`? An HTML list is usually `ul` (**U**nordered **L**ist) or `ol` (**O**rdered **L**ist) and `li` the **L**ist **I**tems themselves.

Comment (josueetcom) [reply portfolio/src/main/webapp/script.js]: This question wasn't answered :)

Comment (TengkuHanaShafikah) [reply portfolio/src/main/webapp/script.js]: Ah, so sorry about that! I made it an unordered list so that the comments don't have bullet points.

Comment (josueetcom) [reply portfolio/src/main/webapp/script.js]: Ah, in that case I think it would've been better to create a `div` with a specific `class` rather than `ul`.

Comment (TengkuHanaShafikah) [reply portfolio/src/main/webapp/script.js]: Oh, that makes sense! I've made the adjustment and will include it in other PR. 👍

**Pull request merged**